apps/main/src/loan/entities/scrvusd-yield.ts

```diff
@@ -3,15 +3,16 @@ import type { Yield } from '@curvefi/prices-api/savings/models'
 import { queryFactory } from '@ui-kit/lib/model/query'
 import { timeOptionValidationSuite } from '@ui-kit/lib/model/query/time-option-validation'
 import type { TimeOption } from '@ui-kit/lib/types/scrvusd'
+import { DAY_IN_MS } from '@ui-kit/themes/design/0_primitives'
 
 export type ScrvUsdYieldWithAverages = Yield & { proj_apy_7d_avg: number; proj_apy_total_avg: number }
 
 export const _getScrvUsdYield = async (params: { timeOption: TimeOption }) => {
   // calcs starting timestamp
   const timeOptionCalc: Record<TimeOption, number> = {
-    '1M': 30 * 24 * 60 * 60 * 1000, // 30 days
-    '6M': 180 * 24 * 60 * 60 * 1000, // 180 days
-    '1Y': 365 * 24 * 60 * 60 * 1000, // 365 days
+    '1M': 30 * DAY_IN_MS, // 30 days
```

Comment:
We're all guilty of it, but honestly we should use `TIME_FRAMES` from `curve-ui-kit\src\lib\model\time.ts` and add missing ones 🙈